docs/the-new-architecture/pillars.md

```diff
@@ -3,8 +3,27 @@ id: pillars
 title: What Compose the New Architecture
 ---
 
-This section recalls the main pillars from the new-architecture-intro section.
+The new architecture is composed mainly by two pillars:
```

Comment:
As a rule of thumb, let's use New Architecture (title case) everywhere as it's like the name of a product